playbooks/gce/openshift-cluster/tasks/launch_instances.yml

```diff
@@ -20,22 +20,57 @@
       - host-type-{{ type }}
       - sub-host-type-{{ g_sub_host_type }}
       - env-host-type-{{ cluster }}-openshift-{{ type }}
-  when: instances |length > 0 
+  when: instances |length > 0
   register: gce
 
+- debug: var=gce
+
+- set_fact:
+    node_label:
+      # There doesn't seem to be a way to get the region directly, so parse it out of the zone.
+      region: "{{ gce.zone | regex_replace('^(.*)-.*$', '\\\\1') }}"
```

Comment:
Fail at this step while deploying with 0 infra node.
In fact we skip the launch of instead in the code above, however this code is executed for the infra.
We should check again that `instances|length >0`